resources/definitions/fdmprinter.def.json

```diff
@@ -2907,6 +2907,16 @@
                     "enabled": "extruders_enabled_count > 1 and machine_nozzle_temp_enabled",
                     "settable_per_mesh": false,
                     "settable_per_extruder": true
+                },
+                "material_alternate_walls":
+                {
+                    "label": "Alternate Wall Directions",
+                    "description": "Alternate wall directions every other layer and inset. Useful for materials that can build up stress, like for metal printing.",
+                    "type": "bool",
+                    "default_value": false,
+                    "enabled": true,
+                    "settable_per_mesh": true,
+                    "settable_per_extruder": true
```

Comment:
According to the ticket this setting should be experimental at first

Reply:
https://github.com/Ultimaker/Cura/commit/0218f77b3cc01793efb43be6a1470443cfc2c3a3